agreements/employment.md

```diff
@@ -0,0 +1,167 @@
+    2. You must always maintain a suitable bank or building society account into which payment can be made, and provide us with current details of it. Changes to your bank or building society details must be notified to us in writing.
+    3. We reserve the right to deduct an amount from any payment due to use from us, or to recover the amound directly from you, if:
+        1. there is an overpayment of wages, expenses or any other sum paid to you during your employment;
+        2. you are paid expenses which are subsequently disallowed;
+        3. there is an outstanding amount due on any loan or advance paid to you by us;
+        4. we become liable for any damage, loss of property, fine or charge as a result of your negligence, dishonesty, breach of contract or failure to follow the company's policies.
+
+4. ### Hours of work
+    1. Unless you work part-time, your normal working day is seven hours long, making a 35 hour working week. During busy periods, you may be required to work longer hours, up to a maximum of 50 hours a week, on average, over a rolling six week period.
+    2. Your normal working hours are 10:00am to 6:00pm with an hour off for lunch. We are open to arrangements for flexible working hours, at the company's sole discretion.
+    3. We do not operate any overtime scheme nor have any arrangements for the paying of bonuses.
+
+5. ### Variable part-time staff
+    1. If your pay is specified as being *variable part time* then you will work for a number days per week to be agreed by the end of business on Wednesday of the previous week.
+    2. You will be paid pro rata as follows: the number of days you have worked each month will be divided by the number of weekdays (Monday to Friday) in the same month, the resulting fraction will be multiplied by one twelfth of the equivalent full time annual salary to give the amount of pay you are entitled to in that month.
+
+6. ### Fixed part-time staff
+    1. If your pay is specified as being "fixed part time" then you will work for a number days per week to be agreed at the commencement of this contract.
+    2. You will be paid pro rata as follows: the number of days you work each week will be divided by five, the resulting fraction will be multiplied by one twelfth of the equivalent full time annual salary to give the amount of pay you are entitled to each month.
+
+7. ### Probationary period
+    1. You will be subject to an initial four month probationary period.
+    2. If at the conclusion of your probationary period the company requires more time to assess your performance, your probationary period may be extended by a further two months.
+
+8. ### Notice period
+    1. In cases of gross misconduct, you may be dismissed without notice. The following is an indicative, non-exhaustive list of things that might constitute gross misconduct:
+        1. insubordination;
+        2. committing a tort or being convicted for a criminal offence that impacts on your employment;
```

Comment:
(still not lawyering) It's a technical term - i.e. with a specific set of meanings. I guess "civil wrong" would work instead, but I doubt it would be any clearer. It's quite possible that dxw doesn't care about this technical meaning and something like "unlawful act" would be clearer and just as useful.